users/userRouter.js

```diff
@@ -0,0 +1,114 @@
+const express = require("express");
+
+const router = express.Router();
+
+const userDb = require("../data/helpers/userDb");
+
+module.exports = router;
+
+const allCaps = (req, res, next) => {
```

Comment:
That middleware appears again in similar fashion to posts. It would be better to make a separate file with middleware and export it to both posts and users and use it like that for more modularity.